src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav {
+  display: block;
+  unicode-bidi: isolate;
```

Comment:
The `unicode-bidi: isolate;` property is used for controlling the text direction and embedding levels. Ensure that this property is necessary for your layout, as it might not have any visible effect in this context.